libs/react-hooks/src/currency/useCurrencyAddressBalance.tsx

```diff
@@ -0,0 +1,17 @@
+import { useWebContext } from '@webb-tools/api-provider-environment';
+import { useMemo } from 'react';
+import { Currency } from '@webb-tools/abstract-api-provider';
+import { useCurrencyBalance } from '@webb-tools/react-hooks';
+
+export function useCurrencyAddressBalance(address: string) {
+  const { apiConfig } = useWebContext();
+  const currency = useMemo(() => {
+    const currencyConfig = apiConfig.getCurrencyByAddress(address);
+    if (!currencyConfig) {
+      return undefined;
+    }
+    return new Currency(currencyConfig);
+  }, [apiConfig, address]);
+  const balance = useCurrencyBalance(currency);
+  return balance ?? '0';
+}
```

Comment:
This doesn't look like its being used anywhere?

Reply:
Will remove it, it's no logger being used